server/api/25words.js

```diff
@@ -0,0 +1,78 @@
+const express = require("express");
+const router = express.Router();
+const sequelize = require("sequelize");
+const { Word, Wordpack } = require("../db");
+
+// GET localhost:3000/api/25words
+router.post("/", async (req, res, next) => {
+  //  find which pack users select and put all the candidate words in an array
+  const { selectedWordPackId } = req.body;
+  const allWords = await Word.findAll({
+    where: {
+      //findAll can work with an array
+      wordpackId: selectedWordPackId,
+    },
+  });
+  //a function to get "quantity" of unique random interger, from 0 - max (inclusive)
+  function getRandomInt(quantity, max) {
```

Comment:
I think it would be good to change the name of this function as well as the comment - it returns an array of size `quantity`, where each element is a random, but unique integer, from 0 - `max` inclusive